test/Microsoft.Azure.Devices.Edge.Test.Common/config/BaseModuleConfigBuilder.cs

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Config
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.Linq;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public abstract class BaseModuleConfigBuilder : IModuleConfigBuilder
+    {
+        IDictionary<string, object> DesiredProperties { get; }
```

Comment:
these should be fields

Reply:
I guess you're right. With the exception of `Name`, these properties are only used internally. I'll change them.

Reply:
Fixed